deployer/billing.py

```diff
@@ -0,0 +1,210 @@
+"""
++TODO: Validate billing export is setup for all clusters we 'manage'
++TODO: Write a JSON Schema for the billing source of truth
++TODO: Differentiate between billing accounts we should *invoice* and ones we don't need to *invoice*
++TODO: Write documentation on when to use this and who uses it
```

Comment:
Should these TODOs be moved into issues under the main billing story rather than in the code?